bazel/repository_locations.bzl

```diff
@@ -1004,8 +1004,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "WebAssembly for Proxies (C++ SDK)",
         project_desc = "WebAssembly for Proxies (C++ SDK)",
         project_url = "https://github.com/proxy-wasm/proxy-wasm-cpp-sdk",
-        version = "fd0be8405db25de0264bdb78fae3a82668c03782",
-        sha256 = "c57de2425b5c61d7f630c5061e319b4557ae1f1c7526e5a51c33dc1299471b08",
+        version = "011ced83a6995167147c00d5baf40a7c356ceefa",
```

Comment:
Update after https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/pull/132 is merged.